models/migrations/v218.go

```diff
@@ -37,8 +38,14 @@ func (*improveActionTableIndicesAction) TableIndices() []*schemas.Index {
 
 	actUserIndex := schemas.NewIndex("au_r_c_u_d", schemas.IndexType)
 	actUserIndex.AddColumn("act_user_id", "repo_id", "created_unix", "user_id", "is_deleted")
-
-	return []*schemas.Index{actUserIndex, repoIndex}
+	indices := []*schemas.Index{actUserIndex, repoIndex}
+	if setting.Database.UsePostgreSQL {
```

Comment:
Wait, so existing postgres instances will never get this performance update?

Reply:
Sync will do it

Reply:
But 218 is already included in 1.17.0?

Reply:
sync run on every start ...

Reply:
Ah, I thought only the migrations that previously did not run are run.

Reply:
You're right migrations that have already run won't be run but Sync() is run every time and that will sync the indices.

The migration was always unnecessary